app/controllers/api/integrations_controller.rb

```diff
@@ -204,18 +199,18 @@ def message_bus?(proxy)
     proxy.oidc? && ZyncWorker.config.message_bus
   end
 
-  def hide_for_apiap
-    raise ActiveRecord::RecordNotFound if apiap?
+  def hide
```

Comment:
I think this is not needed anymore, you might want to remove the method entirely,

Reply:
Ok, I didn't see the #TODO and comments